app/templates/components/hyper-table.hbs

```diff
@@ -27,12 +27,14 @@
   </div>
 
   <div class="right-side">
-    <button
-      class="margin-right-xx-sm margin-bottom-xxx-sm upf-btn upf-btn--default upf-btn--small"
-      data-control-name="table_reset_filters"
-      {{action "resetAllFilters"}}>
-      Reset Filters
-    </button>
+    {{#if manager.options.features.filtering}}
+      <button
+        class="margin-right-xx-sm margin-bottom-xxx-sm upf-btn upf-btn--default upf-btn--small"
+        data-control-name="table_reset_filters"
+        {{action "resetAllFilters"}}>
+        Reset Filters
```

Comment:
You know what I am going to say: can you add to transalation ? :)

Reply:
leave me alone, sir !